apps/full-site-editing/full-site-editing-plugin/full-site-editing/tests/e2e/fse-back-button.spec.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+/* eslint-disable import/no-extraneous-dependencies */
+import { createNewPost } from '@wordpress/e2e-test-utils';
+
+describe( 'Full Site Editing Back Button', () => {
+	it( 'Should navigate to a new post.', async () => {
```

Comment:
Im a little confused on what we are testing here.  Why should the FSE back button navigate to a new post?  

I am guessing `createNewPost()` leads us to that page, but we are just checking the title and not anything with the Back Button itself?  Is this test currently a placeholder to ensure the suite is running properly or something?

Reply:
Yes, it is just a placeholder :) Originally I was going to go into more depth, but I felt we could wait to go into depth after we merge this. So I should probably rename that